nodestream/pipeline/value_providers/normalizer_value_provider.py

```diff
@@ -13,7 +13,11 @@ def __init__(self, using: str, data: ValueProvider):
         self.data = data
 
     def single_value(self, context: ProviderContext) -> Any:
-        return next(self.many_values(context))
+        values = list(self.many_values(context))
+        if values:
+            return next((value for value in values))
+        else:
+            return None
```

Comment:
```suggestion
        return next(self.many_values(context), default=None)
```

I think this is a cleaner solution

Reply:
Yeah thats way cleaner didn't even notice that was an option.